easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-2.9.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'MakeCp'
+
+name = 'HiC-Pro'
+version = '2.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://nservant.github.io/HiC-Pro'
+description = """HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data) 
+ to the normalized contact maps."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/nservant/HiC-Pro/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['b82f15747bce937c3337b8120ad331a4bc07bbed6708d1d38b7d6ff1047a7a81']
+
+# buildopts=" configure" 
```

Comment:
@timeu - any reason why this comment is still here? Something you found during testing?

Reply:
no I think it was an artifact from a copy. I will remove it.